runtimes/python/versions/latest/src/server.py

```diff
@@ -162,30 +145,25 @@ async def execute(context):
     return resp
 
 
-@app.route("/", defaults={"u_path": ""}, methods=HTTP_METHODS)
-@app.route("/<path:u_path>", methods=HTTP_METHODS)
-async def handler(u_path):
+async def handler(request) -> web.Response:
     logger = Logger(
         request.headers.get("x-open-runtimes-logging", ""),
         request.headers.get("x-open-runtimes-log-id", ""),
     )
 
     try:
         return await action(logger, request)
+    except web_exceptions.HTTPClientError as e:
+        return e
```

Comment:
What is this doing exactly, do we have specific use case?

I am worried it could expose some data to client that's meant for developer.